profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -382,14 +370,38 @@ function _tmgmt_poetry_form_tmgmt_job_submit($form, &$form_state) {
   }
 
   // Find unchecked translations to be removed.
-  $result = _tmgmt_poetry_obtain_related_translation_jobs($language_targets_to_remove, 'SUB_' . $origin_job->tjid);
-  // If any, removed found translations.
-  if ($result) {
-    foreach ($result as $row) {
-      // Remove if the target language is the same as the main translation.
-      if ($origin_job->tjid !== $row->tjid && $origin_job_item->tjiid !== $row->tjiid) {
-        entity_get_controller('tmgmt_job')->delete(array($row->tjid));
-        entity_get_controller('tmgmt_job_item')->delete(array($row->tjiid));
+  // This is usefull for the #cart (but it was here before).
```

Comment:
True, but what we had here before didn't make any sense (it was done by @imanoleguskiza long time ago and I checked with him). I took a lot of time to understand this, I had to, now it is a good opportunity for improvement. Like this, we are improving the existing code instead of leaving the nonsense here and adding more code anywhere else.
Of course we have a small risk (I say small because tests are passing still), but it's worth to take it.